qutip/tests/solver/test_stochastic.py

```diff
@@ -380,3 +414,91 @@ def test_small_step_warnings(method):
             qeye(2), basis(2), [0, 0.0000001], [qeye(2)],
             options={"method": method}
         )
+
+
+@pytest.mark.parametrize("method", ["euler", "platen"])
+@pytest.mark.parametrize("heterodyne", [True, False])
+def test_run_from_experiment_close(method, heterodyne):
+    N = 10
+
+    H = num(N)
+    a = destroy(N)
+    sc_ops = [a, a.dag() * 0.1]
```

Comment:
Don't these `sc_ops` lead to basically the same `m_ops`?

Reply:
I see. Would it make sense to add one more version of the test including some other `sc_op`, for example a Hermitian one?